diff --git a/CMakeLists.txt b/CMakeLists.txt
index b7ee3687e..5fe5406a8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -602,10 +602,8 @@ add_compile_definitions(QAPPLICATION_CLASS=QApplication)
 # Installation
 
 install(TARGETS QOwnNotes DESTINATION bin)
-install(DIRECTORY images/icons DESTINATION share/icons/hicolor)
+foreach(SIZE 16x16 24x24 32x32 48x48 64x64 96x96 128x128 256x256 512x512 scalable)
+    install(DIRECTORY images/icons/${SIZE} DESTINATION share/icons/hicolor)
+endforeach()
 install(FILES PBE.QOwnNotes.desktop DESTINATION share/applications)
-if(QON_QT6_BUILD)
-  install(FILES ${QON_QM_FILES} DESTINATION share/qt6/translations)
-else()
-  install(FILES ${QON_QM_FILES} DESTINATION share/qt5/translations)
-endif()
+install(FILES ${QON_QM_FILES} DESTINATION share/QOwnNotes/translations)
diff --git a/QOwnNotes.pro b/QOwnNotes.pro
index c3de6baab..e416756b6 100644
--- a/QOwnNotes.pro
+++ b/QOwnNotes.pro
@@ -427,11 +427,7 @@ unix {
       desktop.files += PBE.QOwnNotes.desktop
   }
 
-  lessThan(QT_MAJOR_VERSION, 6) {
-      i18n.path = $$DATADIR/qt5/translations
-  } else {
-      i18n.path = $$DATADIR/qt6/translations
-  }
+  i18n.path = $$DATADIR/QOwnNotes/translations
   i18n.files += languages/*.qm
 
   icons.path = $$DATADIR/icons/hicolor
diff --git a/main.cpp b/main.cpp
index 28832404c..b679519f8 100644
--- a/main.cpp
+++ b/main.cpp
@@ -52,13 +52,8 @@ void loadTranslations(QTranslator *translator, const QString &locale) {
     loadTranslation(translator[5], appPath + "/languages/QOwnNotes_" + locale);
     loadTranslation(translator[6], appPath + "/QOwnNotes_" + locale);
     loadTranslation(translator[7], "../src/languages/QOwnNotes_" + locale);
-#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
-    loadTranslation(translator[8], "../share/qt6/translations/QOwnNotes_" + locale);
-    loadTranslation(translator[9], appPath + "/../share/qt6/translations/QOwnNotes_" + locale);
-#else
-    loadTranslation(translator[8], "../share/qt5/translations/QOwnNotes_" + locale);
-    loadTranslation(translator[9], appPath + "/../share/qt5/translations/QOwnNotes_" + locale);
-#endif
+    loadTranslation(translator[8], "../share/QOwnNotes/translations/QOwnNotes_" + locale);
+    loadTranslation(translator[9], appPath + "/../share/QOwnNotes/translations/QOwnNotes_" + locale);
     loadTranslation(translator[10], "QOwnNotes_" + locale);
 }
 
@@ -67,11 +62,7 @@ void loadTranslations(QTranslator *translator, const QString &locale) {
  */
 inline void loadReleaseTranslations(QTranslator &translatorRelease, const QString &locale) {
     loadTranslation(translatorRelease,
-#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
-                    "/usr/share/qt6/translations/"
-#else
-                    "/usr/share/qt5/translations/"
-#endif
+                    "/usr/share/QOwnNotes/translations/"
                     "QOwnNotes_" +
                         locale);
 }
